packages/orchestration/src/orchestration-types.ts

```diff
@@ -166,3 +166,42 @@ export interface DocumentGroundingServiceConfig {
    */
   output_param: string;
 }
+
+/**
+ * Filter configuration for Azure content safety Filter.
+ */
+export interface AzureContentFilter {
+  /**
+   * The filter category for hate content.
+   */
+  Hate?: AzureFilterThreshold;
+  /**
+   * The filter category for self-harm content.
+   */
+  SelfHarm?: AzureFilterThreshold;
+  /**
+   * The filter category for sexual content.
+   */
+  Sexual?: AzureFilterThreshold;
+  /**
+   * The filter category for violence content.
+   */
+  Violence?: AzureFilterThreshold;
+}
+
+/**
+ * A descriptive constant for Azure content safety filter threshold.
+ * @internal
+ */
+export const azureFilterThreshold = {
```

Comment:
while this is allowed and technically correct, I am think maybe having slightly different names would make it less confusing.

Reply:
Renaming to `AzureFilterThresholdLevel` as I can't think of a better name.
But honestly these names are very close to `AzureContentSafety` and `AzureThreshold`, the generated types, so it will probably help if we can add some suffix to know that they are custom types. But, I don't have a good idea here.

But, since these are types are not exported anwyay, it should be okay right? Only that we might get confused between the types.

Reply:
for the const you can use supportedAzureThresholds?

Reply:
or supportedAzureFilterThresholds

Reply:
I picked `azureFilterThreshold` to align [with Java,](https://github.com/SAP/ai-sdk-java/pull/169/files#diff-8842d71932fe24a1ef98d191462879e95cb9b8620dc82eca573af248df28231d), but since we don't export the constant anyway, I am okay with renaming.

Reply:
The type is now called `AzureFilterThreshold` and the const is called `supportedAzureFilterThresholds`.
The const is bit wordy, but since it's not exported, I think that's okay.